.github/ISSUE_TEMPLATE/documentation_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Documentation
+about: Ticket for when documentation needs to be changed, updated, or added
+title: Title of the documentation task
+labels: documentation
+assignees: ''
+
+---
+
+**Documentation Request Type**
+<!-- if there's more than one request type, please consider adding multiple tickets -->
```

Comment:
I'm just going to comment on this for now until I know that I can suggest a commit without breaking everything. Suggestion: `For multiple requests, please submit one ticket per issue.`

Reply:
done